neuraxle/steps/output_handlers.py

```diff
@@ -132,15 +134,37 @@ def handle_inverse_transform(self, data_container: DataContainer, context: Execu
 
     def _set_expected_outputs(self, data_container, new_expected_outputs_data_container) -> DataContainer:
         if len(data_container.data_inputs) != len(data_container.expected_outputs):
-            raise AssertionError('OutputTransformerWrapper: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs.')
+            raise AssertionError(
+                'OutputTransformerWrapper: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs.')
 
         data_container.set_expected_outputs(new_expected_outputs_data_container.data_inputs)
         data_container.set_current_ids(new_expected_outputs_data_container.current_ids)
 
         return data_container
 
 
-class InputAndOutputTransformerWrapper(ForceHandleOnlyMixin, MetaStep):
+class _DidProcessInputOutputHandlerMixin:
+    def _did_process(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        di, eo = data_container.data_inputs
+        if len(di) != len(eo):
+            raise AssertionError(
+                '{}: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs.'.format(
```

Comment:
```suggestion
                '{}: Found different len for data inputs, and expected outputs. Please return the same the same amount of data inputs, and expected outputs, or otherwise create your own handler methods to do more funky things.'.format(
```